sources/packages/web/src/services/AuthService.ts

```diff
@@ -95,6 +95,7 @@ export class AuthService {
           this.renewTokenIfExpired,
           RENEW_AUTH_TOKEN_TIMER,
         );
+
```

Comment:
Avoiding committing if there are no changes, unless it is a default prettier format 😉